protons/calibration.py

```diff
@@ -421,17 +370,17 @@ class Tyrosine(Lysine):
     """
     Amber constant-pH TYR residue state weights at given pH
     """
-    pKa = 9.6
+    pka = 9.6
 
 
 class Cysteine(Lysine):
     """
     Amber constant-pH CYS residue state weights at given pH
     """
-    pKa = 8.5
+    pka = 8.5
 
 
-class CalibrationSystem(object):
+class AmberCalibrationSystem(object):
```

Comment:
Yes, thanks for pointing this out. It should not be too hard to generalize this, but for now it wasn't necessary.